### PR TITLE
Manually merge 2.10.x into 2.11.x

### PR DESCRIPTION
**Pull request by morozov**
On top of the changes from `2.10.x`, contains the removal of the handling of the obsolete performance test group.

### CODE REVIEW

**Comment by morozov** [comment]
@greg0ire this assumes that you finished working on the code coverage fixes in `2.10.x`. Otherwise, the merge up can wait.

**Comment by greg0ire** [comment]
Let me just rebase #4180

**Review by greg0ire — DISMISSED** [review]
I have to go, approving in case the results of #4180 look ok to you

File: ci/github/phpunit.oci8.xml [hunk 1]
```diff
@@ -33,10 +33,4 @@
             <directory suffix=".php">../../lib</directory>
         </whitelist>
     </filter>
-
-    <groups>
-        <exclude>
-            <group>performance</group>
-        </exclude>
-    </groups>
```
Comment (greg0ire) [inline ci/github/phpunit.oci8.xml:41]: Also, where does this come from?

Comment (morozov) [reply ci/github/phpunit.oci8.xml:41]: See https://github.com/doctrine/dbal/pull/3923.

**Comment by morozov** [comment]
> […] approving in case the results of #4180 look ok to you.

This time, the coverage of a couple of OCI8-related classes dropped. Supposedly this is what integration tests would cover. Looks legit.

**Review by greg0ire — APPROVED** [review]

**Pull request merged**